schemas/score-v1b1.json

```diff
@@ -72,133 +89,154 @@
+          "type": "object",
+          "additionalProperties": true
         }
       }
     },
     "resourcesLimits": {
-      "description": "The compute resources limits.",
+      "description": "The compute and memory resource limits.",
       "type": "object",
-      "minProperties": 1,
       "additionalProperties": false,
       "properties": {
         "memory": {
-          "description": "The memory limit.",
-          "type": "string"
+          "description": "The memory limit in bytes with optional unit specifier. For example 125M or 1Gi.",
+          "type": "string",
+          "pattern": "^[1-9]\\d*(K|M|G|T|Ki|Mi|Gi|Ti)?$"
         },
         "cpu": {
-          "description": "The CPU limit.",
-          "type": "string"
+          "description": "The CPU limit as whole or fractional CPUs. 'm' indicates milli-CPUs. For example 2 or 125m.",
+          "type": "string",
+          "pattern": "^\\d*(?:m|\\.\\d+)?$"
         }
       }
     },
     "container": {
-      "description": "The container name.",
+      "description": "The specification of a Container within the Workload.",
       "type": "object",
       "required": [
         "image"
       ],
       "additionalProperties": false,
       "properties": {
         "image": {
-          "description": "The image name and tag.",
-          "type": "string"
+          "description": "The container image name and tag.",
+          "type": "string",
+          "minLength": 1
         },
         "command": {
-          "description": "If specified, overrides container entry point.",
+          "description": "If specified, overrides the entrypoint defined in the container image.",
           "type": "array",
-          "minItems": 1,
           "items": {
             "type": "string"
           }
         },
         "args": {
-          "description": "If specified, overrides container entry point arguments.",
+          "description": "If specified, overrides the arguments passed to the container entrypoint.",
           "type": "array",
-          "minItems": 1,
           "items": {
             "type": "string"
           }
         },
         "variables": {
           "description": "The environment variables for the container.",
           "type": "object",
-          "minProperties": 1,
+          "propertyNames": {
+            "minLength": 1,
+            "pattern": "^[^=]+$"
+          },
           "additionalProperties": {
             "type": "string"
           }
         },
         "files": {
-          "description": "The extra files to mount.",
+          "description": "The extra files to mount into the container.",
           "type": "array",
-          "minItems": 1,
           "items": {
             "type": "object",
+<<<<<<< HEAD
             "required": [
               "target"
             ],
+=======
+            "required": [
+              "target"
+            ],
+>>>>>>> 95847dd8aa805318cb287c6d25dcddf965f50c26
```

Comment:
Merge relict?

Reply:
fixed up! yup looks like some undetected merge conflict. I've re-copied the schema in from source.